### DISCUSSION TITLE
Cara, primeiramente parabéns!

### DISCUSSION THREAD

**Discussion by justynik** [lucasvieceli/bot-electron#4] [General]
Você foi o primeiro até então a estar desenvolvendo uma interface gráfica open source, user friendly, e com maestria, gostei de ver o seu trabalho e espero que continue da forma que estava, força amigo! Não sou nenhum bom programador, apenas um entusiasta da tecnologia e programação, mas se precisar de qualquer coisa e estiver ao meu alcance ajuda-lo, o farei. Até breve, abraços.

**Comment by lucasvieceli** [comment]
eu parei porque realmente não estava compensando o esforço, mas estou com outras ideias de projeto caso tenha interesse, chama no telegram @lucasvieceli

**Comment by lucasvieceli** [comment]
@justynik  hj vou abrir esse projeto e corrigi o login pra pelo menos funcionar o bot, irei testar pra ver quanto de memoria ele ocupa quando tiver rodando com 20 contas
